day3/main.go

```diff
@@ -79,7 +79,7 @@ type slopeLine struct {
 }
 
 func (s *slopeLine) isTree(pos int) bool {
-	if pos > s.length {
+	if pos >= s.length {
```

Comment:
This is all that matters. 
Excuse the line changes and formatting changes. 

But I also deleted your binary which seems like a net positive.

Reply:
awesome!